pipelines/streamdiffusionv2/modular_blocks.py

```diff
@@ -18,15 +19,20 @@
 
 logger = diffusers_logging.get_logger(__name__)
 
-# Main pipeline blocks for V2V workflow
-ALL_BLOCKS = InsertableDict(
+# Block sequences for StreamDiffusionV2
+# Video mode: consumes input video and uses motion-aware noise control.
+VIDEO_BLOCKS = InsertableDict(
     [
         ("text_conditioning", TextConditioningBlock),
         ("embedding_blending", EmbeddingBlendingBlock),
         ("set_timesteps", SetTimestepsBlock),
         ("preprocess_video", PreprocessVideoBlock),
         ("noise_scale_controller", NoiseScaleControllerBlock),
         ("setup_caches", SetupCachesBlock),
+        # PrepareLatentsBlock always runs to create base latents for text-to-video
```

Comment:
See other comment about using AutoPipelineBlocks. Maybe that would result in less duplication since there would just be a conditional switch on presence of an input `video` to use PrepareLatentsBlock vs. PrepareVideoLatentsBlock.